src/IdTokenResponse.php

```diff
@@ -55,17 +55,10 @@ protected function getBuilder(
             ($this->useMicroseconds ? microtime(true) : time())
         );
 
-        if ($this->currentRequestService) {
-            $uri = $this->currentRequestService->getRequest()->getUri();
-            $issuer = $uri->getScheme() . '://' . $uri->getHost() . ($uri->getPort() ? ':' . $uri->getPort() : '');
-        } else {
-            $issuer = 'https://' . $_SERVER['HTTP_HOST'];
-        }
-
         return $this->config
             ->builder()
             ->permittedFor($accessToken->getClient()->getIdentifier())
-            ->issuedBy($issuer)
+            ->issuedBy('https://' . $_SERVER['HTTP_HOST'])
```

Comment:
I don't think we should force the https here, we could use the default value from laravel and fallback on https only if missing. 
Locally, we may want to work on unsecured connection.

Reply:
To be fair it was `'https://' . $_SERVER['HTTP_HOST']` before https://github.com/jeremy379/laravel-openid-connect/pull/13